src/wycheproof/WYCHEPROOF_COPYING

```diff
@@ -0,0 +1,212 @@
+* The file `ecdsa_secp256k1_sha256_bitcoin_test.json` in this directory
+  comes from Google's project Wycheproof with git commit
+  `b063b4aedae951c69df014cd25fa6d69ae9e8cb9`.
+  The Wycheproof project can be found at https://github.com/google/wycheproof
```

Comment:
In the spirit of @Sajjon 's suggestion:

```suggestion
* The file `ecdsa_secp256k1_sha256_bitcoin_test.json` in this directory
  comes from Google's project Wycheproof with git commit
  `b063b4aedae951c69df014cd25fa6d69ae9e8cb9`, see
  https://github.com/google/wycheproof/blob/b063b4aedae951c69df014cd25fa6d69ae9e8cb9/testvectors_v1/ecdsa_secp256k1_sha256_bitcoin_test.json .
```

I think this is useful, I first looked in the wrong directory (`testvectors` instead of the new `testvectors_v1`...).